app/assets/stylesheets/style.scss

```diff
@@ -31,7 +31,10 @@ label {
   color: white;
 }
 #dark-panel {
-  background:rgba(35,56,63,0.95);;
+  background:rgba(31,138,150,0.95);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space